Docs: Add configuration option for the image renderer (#41798)
grafana/grafana: docs/sources/administration/configuration.md

### rendering_clustering_max_concurrency

> When rendering_mode = clustered you can define the maximum number of browser instances/incognito pages that can execute concurrently. Default is `5`.

Reviewer: ```suggestion
When rendering_mode = clustered, you can define the maximum number of browser instances/incognito pages that can execute concurrently. Default is `5`.
```

### rendering_clustering_timeout

> **Note**: Available in grafana-image-renderer v3.3.0 and later versions.

> When rendering_mode = clustered you can instruct how long a rendering request can take before timing out. Default is `30` seconds.

Reviewer: ```suggestion
When rendering_mode = clustered, you can specify the duration a rendering request can take before it will time out. Default is `30` seconds.
```
